modyn/selector/internal/selector_strategies/downsampling_strategies/gradnorm_downsampling_strategy.py

```diff
@@ -0,0 +1,6 @@
+from modyn.selector.internal.selector_strategies.downsampling_strategies import AbstractDownsamplingStrategy
+
+
+class GradNormDownsamplingStrategy(AbstractDownsamplingStrategy):
+    def get_downsampling_strategy(self) -> str:
+        return "RemoteGradNormDownsampling"
```

Comment:
maybe we should add a comment somewhere (parent class) that this is the name of the identifier of the class that will be instantiated on the trainer server. goes together with my comment this should be named identifier or sth

Reply:
is that addressed?